gradle/libs.versions.toml

```diff
@@ -15,7 +15,7 @@ agp = "8.5.2"
 # https://github.com/TWiStErRob/net.twisterrob.gradle/releases
 twisterrob = "0.17"
 detekt = "1.23.6"
-dependencyAnalysis = "1.33.0"
+dependencyAnalysis = "2.0.0"
```

Comment:
> Task :generateBuildHealth
No project health reports found. Is 'com.autonomousapps.dependency-analysis' not applied to any subprojects in this build?
See https://github.com/autonomousapps/dependency-analysis-gradle-plugin/wiki/Adding-to-your-project